lib/gettext-setup/gettext_setup.rb

```diff
@@ -19,7 +20,7 @@ def self.initialize(locales_path, options = {})
     @@locales_path = locales_path
 
     # Make the translation methods available everywhere
-    Object.send(:include, FastGettext::Translation)
+    Object.send(:include, FastGettext::Translation, FastGettext::TranslationMultidomain)
```

Comment:
I can understand the other changes to do with locales list, but I didn't see any reference to what this `FastGettext::TranslationMultidomain` does.  Can you add that to the commit message?